test/functional/forall/reduce-multiple-segment/tests/test-forall-segment-multiple-ReduceMin.hpp

```diff
@@ -50,7 +50,7 @@ void ForallReduceMinMultipleTestImpl(IDX_TYPE first,
 
   DATA_TYPE current_min = default_val;
 
-  RAJA::ReduceMin<REDUCE_POLICY, DATA_TYPE> min0;
+  RAJA::ReduceMin<REDUCE_POLICY, DATA_TYPE> min0(0);
```

Comment:
are these tests to test that default reduction arguments work?

Reply:
This is required as the default constructor for omp-target Reduce_Data has been deleted.

Reply:
To clarify, omp-target reductions have a different interface than the others? If this is the case, please add an issue so that we can resolve this inconsistency.

Reply:
Can you also add a comment to the constructors in the test to indicate which should be default.